R/utils.R

```diff
@@ -95,7 +95,7 @@
         assay.type, x, name = .get_name_in_parent(assay.type)){
     if(!.is_non_empty_string(assay.type)){
         stop("'",name,"' must be a single non-empty character value.",
-            call. = FALSE)
+             call. = FALSE)
```

Comment:
I guess this is not 4 spaces anymore. Indentations must be always multiplication of 4 spaces.